motif-docs/styles/main.css

```diff
@@ -152,3 +152,21 @@ code::before, code::after {
 .token.inserted.prefix {
   @apply text-emerald-200 text-opacity-75 select-none;
 }
+
+.prose h2 a {
+  border-bottom-width: 0px !important;
+}
+
+.prose h3 a {
+  border-bottom-width: 0px !important;
+}
+
+.prose h2 a:hover {
+  border-bottom-width: 2px !important;
+}
+
+.prose h3 a:hover {
+  border-bottom-width: 2px !important;
+}
```

Comment:
@dillonraphael did you compare this pr side by side with production docs? I'm still seeing a difference :)

Reply:
@flybayer 🙃